Sample Data/Custom/SenservaPro_CL.json

```diff
@@ -1,158 +1,169 @@
 [
 	{
-		"Date": "2021-03-25T15:11:08.7434776+00:00",
-		"Severity": 3,
-		"ControlName": "TestControl1",
-		"Description": "Test Description",
-		"Id": 74,
-		"Value": "{\"User\":\"Test@testdomain.com\"}",
-		"Reference": "",
-		"Group": "User",
-		"Weight": 500,
-		"Tenant": {
-			"TenantId": "e2e9ace3-5769-4fa8-8536-05e8f850cb52",
-			"BlockBlobUri": "",
-			"AppendBlobUri": "",
-			"CustomerVaultUrl": "",
-			"EventHubNamespaceConnectionString": "",
-			"EventHubName": "",
-			"BestAzureLicenseFound": 0,
-			"SenservaVersion": 0,
-			"Installed": "2020-10-30T16:41:30",
-			"MagicCallBacks": {},
-			"TenantDisplayName": "SenservaTestTenant",
-			"ExportWorkspaceDisplayName": "SenservaTestWorkspace"
-		},
-		"Errors": ""
+		"ControlName": "UserTest",
+		"Value": "[{\"Tag\":\"UserName\",\"Value\":\"TestUser (test@test.com)\"},{\"Tag\":\"Roles\",\"Value\":{\"$values\":[]}}]",
+		"Reference": [],
+		"Group": "Users",
+		"MitreControls": [
+			{
+				"Target": 0,
+				"ComplianceType": "MITRE",
+				"Ids": [
+					"T1098"
+				]
+			}
+		],
+		"NistControls": [
+			{
+				"Target": 0,
+				"ComplianceType": "NIST",
+				"Ids": [
+					"AC-3"
+				]
+			}
+		],
+		"TenantId": "1a99934f-e99c-4bc8-ad3a-740db6132b2b",
```

Comment:
@tj-senserva , Can you please update on this ? Thanks!.

Reply:
@v-maudan The connector sends this data as a string. There should not be an update needed
![image](https://user-images.githubusercontent.com/59487793/120999591-ecd9b800-c74e-11eb-9916-a465f450d716.png)

Reply:
@tj-senserva , Thanks for update, Please update TenantId value in sample data as string which given as GUID format.[PFB for ref.]
"TenantId": "1a99934f-e99c-4bc8-ad3a-740db6132b2b", - GUID format

Reply:
@v-maudan I understand that is the GUID format, I don't understand what you are wanting in the Sample Data though. That is what our connector sends when it converts the GUID to a string, the data type changes but the format is still GUID format

Reply:
@tj-senserva ,Thanks for update, Please update any sample string against "TenantId": so that we will ingest sample data properly in log analytics workspace and workbook queries will not fail. (Currently workbook queries are failing which contains TenantId_s ) Thanks!

Reply:
@v-maudan @shainw I have updated the sample data per request, though this no longer accurately represents what our data connector will send

![image](https://user-images.githubusercontent.com/59487793/121914269-b9bb9980-ccf7-11eb-93c1-ed880ebd4ca6.png)